components/Footer/index.js

```diff
@@ -1,47 +1,72 @@
 import style from "./style.scss";
 import Link from "next/link";
 
+const Contact = () => (
+  <div className={style.contact}>
+    <h2>Contact Us</h2>
+    <h1>contact@warsawex.org</h1>
+  </div>
+);
+const Logo = () => (
+  <div className={style.warsawex_logo}>
+    <img className={style.symbol} src="/static/warsawex-symbol.svg" />
+    <img
+      className={style.logo}
+      src="/static/warsawex-logo-white.svg"
+      alt="warsaw.ex meetup logo white"
+    />
+  </div>
+);
+const Menu = () => (
+  <div className={style.menu}>
+    <Link href="/speak-up">
+      <h2>
+        <a>Speak Up</a>
+      </h2>
+    </Link>
+    <Link href="/about-us">
+      <h2>
+        <a>About Us</a>
+      </h2>
+    </Link>
+  </div>
+);
+const Media = () => (
+  <div className={style.media}>
+    <a href="https://www.facebook.com/warsawex.meetup/" target="_blank">
+      <img src="/static/facebook_white.svg" alt="white Facebook logo" />
+    </a>
+    <a
+      href="https://www.youtube.com/channel/UCJtBwpt8UQQKQ0K0UNNcnJg"
+      target="_blank"
+    >
+      <img src="/static/youtube_white.svg" alt="white Youtube logo" />
+    </a>
+    <a href="http://warsawex-slackin.herokuapp.com" target="_blank">
+      <img src="/static/slack_white.svg" alt="white Slack logo" />
+    </a>
+    <a href="https://www.meetup.com/pl-PL/warsaw-ex/" target="_blank">
+      <img src="/static/meetup_white.svg" alt="white Meetup logo" />
+    </a>
+  </div>
+);
+
```

Comment:
Co sądzisz o wyeksportowaniu tego do osobnych pllików i zaimportowaniu componentów w tym? Wg mnie to może posprzątać trochę plik oraz przyśpieszyć szukanie czegoś w przyszłości. Załóżmy, że chcę zmienić coś w linku do facebooka w footerze - najpierw poszukałbym folderu 'Footer', potem mógłbym od razu przeskoczyć do pliku 'Media' albo zobaczyć jaki jest podział na componenty w index.js i przejść do 'Media'. Zaoszczędza to scrollowania plików :smiley_cat: Wiem, że teraz to nie jest jakiś skomplikowany przypadek, ale wg mnie takie rozbijanie jest dobrym ćwiczeniem przed bardziej skomplikowanymi strukturami, w których scrollowanie może być frustujące i kłopotliwe.

Reply:
Chyba ma to sens ;)